### DISCUSSION TITLE
reference/docs/feds

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#597] [Q&A]
# reference/docs/feds

https://wangchujiang.com/reference/docs/feds.html

**Comment by xlcaptain** [comment]
这个导航的前面能加icon吗？现在是不支持吗？必须跟md文件名一样的svg才可以？

**Reply by jaywcjlove** [reply]
原来想 加 favicon 作为展示，其实可以本地添加图标，主要是我懒得搞
